terraform/system/aws/main.tf

```diff
@@ -130,6 +130,8 @@ module "rack" {
   ssl_protocols       = var.ssl_protocols
   subnets             = module.cluster.subnets
   tags                = local.tag_map
+  telemetry           = var.telemetry
+  telemetry_file      = "${var.settings}/vars.json"
```

Comment:
isn't it better if we call `telemetry_file` as `variables_file` or `settings_file`?